src/command.js

```diff
@@ -2,74 +2,115 @@ import program from 'commander';
 import fs from 'fs';
 import path from 'path';
 import YAML from 'yamljs';
-import { defaultBlockNumber } from './config';
-import { isAddress, validateBlockNumber } from './web3/utils';
+import { defaultBlockNumber, watchingConfigPath, getEnv, saveStateFileName } from './config';
+import { isAddress, getLastBlock } from './web3/utils';
 import { isPathExist } from './utils';
 
 /**
  * convert string to array
- * @param {string} val 
+ * @param {string} val
  * @return Array
  */
 const list = val => val.split(',');
 
 
-export default (filePath, lastBlockNumber) => {
-  const watchConfig = YAML.load(filePath);
-  program
-    .version('0.1.0')
-    .option('-a, --addresses <n>', 'List of address', list)
-    .option('-f, --from [n]', 'From block', typeof watchConfig.from !== 'undefined' ? watchConfig.from : defaultBlockNumber)
-    .option('-t, --to [n]', 'To block', typeof watchConfig.to !== 'undefined' ? watchConfig.to : defaultBlockNumber)
-    .option('-q, --quick [n]', 'Quick Mode', typeof watchConfig.quick !== 'undefined' ? watchConfig.quick : false)
-    .option('-s, --save-state [n]', 'Save state', typeof watchConfig.saveState !== 'undefined' ? watchConfig.saveState : null);
+export class Commands {
```

Comment:
Why did you change it to class?

Reply:
It was a solution to the problem, with a class all variables declared as this have `getters` automatically. do you have a better solution?